Source/Models/Brick.swift

```diff
@@ -9,6 +9,24 @@
 import Foundation
 import UIKit
 
+#if os(iOS)
+public protocol BrickPreviewingDelegate: class {
```

Comment:
Same this is fine to go into tvOS, just add a comment above saying not applicable in tvOS